packages/navitem/src/react/context.tsx

```diff
@@ -12,15 +14,17 @@ export interface ContextValue {
   bar?: ReactNode
   icon?: ReactNode
   menu: boolean
-  renderContainer: (props: ContainerProps) => JSX.Element
+  renderContent: (props: ContentProps) => JSX.Element
```

Comment:
Why the switch from renderContainer?

I know you said you "didn't like" the naming.  

I think I prefer "container" generally.  Because generally we're replacing some outermost tag, in this case the `button` with an `a`, etc.  This also seems a common term for this problem outside this codebase.

In scrollable, I went with `renderContent` because it was the _inner_ tag (the... content 🤷 , I dunno, it's a pretty lame generic term too) that I was changing the tagName of.

Reply:
yeah i just matched the name in scrollable